kernel/packages/build-ecs/tsconfig.json

```diff
@@ -10,7 +10,13 @@
     "emitDecoratorMetadata": true,
     "declaration": false,
     "downlevelIteration": true,
-    "keyofStringsOnly": true
+    "keyofStringsOnly": true,
```

Comment:
I think we should leave out this checks, or at least make them optional somehow (have a clear and easy way to remove them).

Otherwise, scene developers are going to see their code not compiling after an update and are going to freak out:
![image](https://user-images.githubusercontent.com/2048532/94030508-4605e500-fd94-11ea-98a3-01ea5bca5fec.png)